internal/spec/definition.go

```diff
@@ -47,8 +47,10 @@ type TestNRQL struct {
 }
 
 type TestNRQLExpectedResult struct {
-	Key   string `yaml:"key"`
-	Value string `yaml:"value"`
+	Key               string      `yaml:"key"`
+	Value             interface{} `yaml:"value"`
```

Comment:
Using the `any` type allows us to compare the expected value and actual values using native type comparisons (i.e. `3==5`) rather than converting them both to string and doing a string comparison (`"3"=="5"`).

Reply:
I see. Then, can we document this behavior somewhere? I think it is not obvious when you are working on the test definition. For instance, according to [this function](https://github.com/newrelic/newrelic-integration-e2e-action/pull/34/files#diff-79a0456242c011e44009a8d20c0446a7dd16aef7ff7382b972883822e8cfed79R192), booleans will work no matter if they are bools or strings but it is not the same for numbers, is it?

Reply:
Fair point. The reason we currently treat false and "false" as the same is because the NRQL API seems to not return a native boolean, rather it returns a string boolean. How would you like to document this?

Reply:
Some notes could be added to the [README's spec definition](https://github.com/newrelic/newrelic-integration-e2e-action#spec-file-for-the-e2e). Moreover, a comment in the function handling the value (and its type) can be useful too.